META-INF/persistence.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Why do we need this?

Reply:
I have no idea because it's an auto-generated script or did you mean about this file in general?

Reply:
I believe this file is obsolete